AAI/README.md

```diff
@@ -95,6 +90,21 @@ proxies for "Claim Consumption Only" functionality --
 Data holders and data owners should explore their options to decide what best
 fits their needs.
 
+#### Embedded Tokens example and explanation
+
+![embedded](https://github.com/ga4gh/data-security/blob/master/AAI/embedded_Claims_flow.png)
+
+Consider two parties: Google and ELIXIR.  
+
+In this example, Google Passport Clearinghouse makes access decisions based on ELIXIR Assertion Repository information via a chain of brokers that have passed along the Passport Visas in standard GA4GH Passport format where the Passports are signed by different Brokers but the Passport Visas retain the signature from the Passport Visa Issuer. 
+
+The way this chain of brokers and trust is maintained is through "embedded tokens". There are two types of embedded tokens: Embedded Access Tokens and Embedded Document Tokens. 
+
+Embedded Access Tokens are claims in a Broker's token that can then be sent to OTHER brokers' `/userinfo` endpoints for further user claims. 
+
+Embedded Document Tokens are not expected to be used with a provider's `/userinfo` endpoint and will have everything in them to verify their provenance and will have the necessary claims in them already.
```

Comment:
"Embedded Document Tokens cannot be revoked and no `/userinfo` endpoint is provided for them, however they stilll offer a signature that can be used to verify their provenance and always contain the necessary claims in them already."